support-frontend/assets/helpers/contributions.ts

```diff
@@ -501,6 +501,15 @@ const contributionTypeAvailable = (
 		(settings) => settings.contributionType === contributionType,
 	);
 
+const contributionsOnlyAmountsTestName = 'VAT_COMPLIANCE';
+
+const isSubjectToContributionsOnlyAmounts = (
```

Comment:
I wonder if a better name for `isSubjectToContributionsOnlyAmounts` might be something like `countriesWhereWeOfferContributionsOnly`?

Reply:
I like including `countries` in the name as this makes it clear that that's what we're basing this on. 

I also like the removal of the `amounts` bit as that's  implementation details.

My 2¢

Reply:
Suggestions taken on board, I've gone with `isContributionsOnlyCountry`